seqr/utils/elasticsearch/constants.py

```diff
@@ -313,6 +329,13 @@
     'dbnsfp_SIFT_pred': {},
     'StrVCTVRE_score': {'response_key': 'strvctvre'},
 }
+MITO_PREDICTION_FIELDS_CONFIG = {
+    'APOGEE_score': {'response_key': 'apogee'},
+    'hap_defining_variant': {'response_key': 'haplogroup_defining', 'format_value': lambda k: 'Y' if k else None},
```

Comment:
It seems that there is a naming rule for the predictions like `PREDICT-SOURCE`_`PREDICT-NAME`. When we don't use `response_key` to rename the field, it will use `PREDICT-NAME` as the default response key. If we want to get rid of `response_key`, we should name the mito prediction fields in the ES index as `mitimpact_apogee`, `hap_haplogroup_defining`, `mitotip_mitoTIP`, and `hmtvar_hmtVar`. The name `mitimpact_apogee` looks reasonable but the rest names are a little weird. What do you think about these field names? @hanars

Reply:
I think they are fine given that many of our predictor names are weird like that (we have `eigen_Eigen_phred` and `mpc_MPC`). But if you really prefer not to give some of them weird names its okay to use  `response_key` here, its up to you